epcis-context.jsonld

```diff
@@ -0,0 +1,19 @@
+{
+    "@context": {
+        "@version": 1.1,
+        "rdf": "http://www.w3.org/1999/02/22-rdf-syntax-ns#",
+        "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
+        "owl": "http://www.w3.org/2002/07/owl#",
+        "skos": "http://www.w3.org/2004/02/skos/core#",
+        "gs1": "https://gs1.org/voc/",
+        "language": "@language",
+        "value": "@value",
+        "EPCISDocumentCredential": {
+            "@id": "gs1:EPCISDocumentCredential"
+        },
+        "EPCISEventCredential": {
+            "@id": "gs1:EPCISEventCredential"
+        },
+        "@import": "https://ref.gs1.org/standards/epcis/2.0.0/epcis-context.jsonld"
```

Comment:
Not necessary if the EPCIS Document/Event carries its own context, as it's supposed to